docs/server.md

```diff
@@ -0,0 +1,98 @@
+
+If there is an error sending the data, you will get an HTTP error code (4xx or 5xx).
+
+## Setup
+
+### Building
+
+Tranquility Server is not available through Maven, so it must be built to be used.
+
+1. Build an artifact by running `sbt 'project server' universal:packageZipTarball`.
+2. This will produce a tarball in `server/target/universal/tranquility-server-VERSION.tgz`.
+3. Unpack the artifact into your directory of choice by running `tar -xzf server/target/universal/tranquility-server-VERSION.tgz`.
+
+### Configuration
+
+Tranquility Server uses a YAML file for configuration. You can see an example in `conf/server.yaml.example` of the
+tarball distribution. You can start off your installation by copying this example file to `conf/server.yaml`. The
+YAML file has two sections:
+
+1. `dataSources` - per dataSource configuration.
+2. `properties` - general server properties that apply to all dataSources. See "Configuration reference" for details.
+
+The dataSources key should contain a mapping of dataSource name to configuration. Each dataSource configuration
+has two sections:
+
+1. `spec` - a YAML representation of a [Druid ingestion spec](http://druid.io/docs/latest/ingestion/index.html). Note
+that the `ioConfig` is expected to be of type `realtime` but otherwise empty (null firehose, null plumber). This is
+because Tranquility supplies its own firehose and plumber.
+2. `properties` - per dataSource properties. See "Configuration reference" for details.
+
+### Running
+
+If you've saved your configuration into `conf/server.yaml`, run the server with:
+
+```bash
+bin/tranquility-server conf/server.yaml
+```
+
+## Configuration reference
+
+With the exception of the global properties (which must be provided at the top level), most properties can be provided
+at the top level or per-dataSource. If the same property is provided at both levels, the per-dataSource version will
+take predence. This provides a way to set default properties that apply to all dataSources, while still allowing
+customization for certain dataSources.
+
+### Global properties
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`http.port`|Port to listen on.|8200|
+|`http.threads`|Number of threads for HTTP requests.|8|
+|`http.idleTimeout`|Abort connections that have had no activity for longer than this timeout. Set to zero to disable. ISO8601 duration.|PT5M|
+
+### Other properties
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.indexService`|Druid Overlord's service.name|druid/overlord|
+|`druid.partitions`|Number of Druid partitions to create.|1|
+|`druid.replicants`|Number of instances of each Druid partition to create. This is the *total* number of instances, so 2 replicants means 2 tasks will be created.|1|
+|`zookeeper.connect`|ZooKeeper connect string.|none; must be provided|
+|`zookeeper.timeout`|ZooKeeper session timeout. ISO8601 duration.|PT20S|
+|`disco.path`|Curator service discovery path.|/druid/discovery|
```

Comment:
I was less sure about that one as it's called `druid.zk.service.host`, but the zk cluster is not just used for druid stuff- it's also used for internal tranquillity coordination. So I was concerned it would be misleading?